src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -7,7 +7,6 @@
     <WarningsNotAsErrors>CS0618</WarningsNotAsErrors>
     <DefineConstants>$(DefineConstants);NETCORE</DefineConstants>
     <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);FEATURE_WINDOWS_SYSTEM_COLORS;FEATURE_SYSTEM_EVENTS</DefineConstants>
-    <DefineConstants Condition="'$(TargetsUnix)' == 'true'">$(DefineConstants);CORECLR</DefineConstants>
```

Comment:
Just to confirm, these constants were unused prior to this change of adding and using AsRef, correct?

Reply:
Correct.